cloud/google/gc_storage.py

```diff
@@ -57,6 +57,11 @@
       - This option let's the user set the canned permissions on the object/bucket that are created. The permissions that can be set are 'private', 'public-read', 'authenticated-read'.
     required: false
     default: private 
+  headers:
+    description:
```

Comment:
Could you add a ` version_added: x.y.z` here?
@bcoca - what version should @richtier use - 1.9.1?